lib/model/Product.dart

```diff
@@ -468,7 +475,8 @@ class Product extends JsonObject {
       this.labels,
       this.labelsTags,
       this.labelsTagsInLanguages,
-      this.packaging,
+      // TODO: deprecated from 2022-12-16; remove when old enough
+      @Deprecated('Use packagingS field instead') this.packaging,
```

Comment:
```suggestion
      @Deprecated('Use [packagings] field instead') this.packaging,
```

Feel free to leave it with the capital S

Reply:
The square bracket syntax is unfortunately not detected in strings, only in `///` comments.